BA-132: add capacity to schedules calculation

FedericoQuarin opened the pull request.

FedericoQuarin committed:
BA-132: add capacity to schedules calculation

Valenreynoso17 commented on src/main/java/com/barapp/web/model/HorarioPorRestaurante.java:
```diff
@@ -38,4 +36,47 @@ public List<ConfiguradorHorario> getLaborales() {
+                    if (!ch.generarHorarios().contains(reserva.getHorario())) {
+                        reservasInvalidas.add(reserva);
+                    } else {
+                        Optional<Mesa> mesaOpt = mesasPorConfigurador.get(ch).get(reserva.getTipoComida()).stream()
```
reserva.getHorario.getTipoComida

ChortJulio committed:
Remove tipoComida from Reserva model

Se quitó TipoComida del modelo de Reserva ya que se obtiene del Horario

ChortJulio committed:
Update getAll favourite and recently seen restaurants

Ahora los restaurantes vistos recientemente y favoritos de los usuarios sólo devuelven restaurantes en estado HABILITADO o PAUSADO

ChortJulio committed:
Update getAll restaurants by user

Ahora al llamar getAll de Restaurantes en mobile se devuelven sólo restaurantes en estado HABILITADO o PAUSADO

ChortJulio reviewed (approved):
LGTM!
